illustratr/blocks.css

```diff
@@ -36,13 +36,12 @@ Description: Used to style Gutenberg Blocks.
 /* Full Width */
 
 .alignfull {
-	width: calc(100vw - 52px); /* subtract 52px for body padding */
-	max-width: calc(100vw - 52px);
-	margin-left: 0;
-	margin-right: 0;
+	margin-left: calc(50% - 50vw + 26px);
+	margin-right: calc(50% - 50vw + 26px);
+	max-width: 100vw;
```

Comment:
I think this is coming from https://github.com/Automattic/themes/pull/2768/
Otherwise, this looks good to go!